Game/Materials.md

```diff
@@ -0,0 +1,168 @@
+
+##### Exotic Flora
+Plants with truley magical properties.
+
+> Glowbulb, firegrass, noxgreen
+
+---
+
+#### Metal
+Rigid workable materials, making great [Armour](Armour), [Gear](Gear), and [Weapons](Weapons). Generally harvested from deposits in the ground or reclaimed.
+
+##### Basic Metal
+Poorly sourced metal that isn't very useful for working.
+
+> Junk metal melted down, pig iron, poorly sourced metal.
+
+##### Artisan Metals
+Simple unimpressive metals that are built for purpose.
+
+> Refined iron, copper, brass.
+
+##### Exotic Metals
+Complex impressive metals with properties great for working with.
+
+> Steel, mithril, dwarven ingots.
+
+---
+
+#### Leather
+Repurposed skin of creature's, leather makes an excellent material for [Storage](Storage), and [Armour](Armour).
+
+##### Basic Leather
+Simple animal leathers.
+
+> cow leather, deer hide.
+
+##### Artisan Leather
+Hardened leathers.
+
+> boiled leather, treated with epoxy, other treatments. 
+
+##### Exotic Leather
+Exotic creature skins with stranger properties.
+
+> Dragon hide, demons skin.
+
+---
+
+#### Fabric
+Thin strands of material carefully assembled into weaves, fabric makes an exceptional material for [Storage](Storage), and [Armour](Armour).
+
+##### Basic Fabric
+Basic poorly made fabrics.
+
+> Simple spotty cotton, recycled fabrics.
+
+##### Artisan Fabric
+Properly made fabric weaves.
+
+> Proper cotton, linens.
+
+##### Exotic Fabric
+High-quality fabrics that have then been treated with additional processes and other exotic weaves.
+
+> Linothorax, quilted weaves, silks, gossamer.
+
+---
+
+#### Wood
+Sturdy natural resource generally reclaimed or harvested from trees, useful for the construction of some [Weapons](Weapons), and [Assets](Assets).
+
+##### Basic Wood
+High-quality wood that can be worked (anything worse than a hard wood isn't useful for crafting).
+
+> Hard woods
+
+##### Artisan Wood
+High-quality wood that has been further treated with alchemical processes.
+
+> Varnished, epoxied.
+
+##### Exotic Wood
+Magical wood, from ancient forests.
+
+> Elven wood, spirit wood, living logs.
+
+---
+
+#### Resonant
+Items with a clear magical attunement, pieces of magical items, mundane items collected from places of power and rare gemstones. Useful in powering [Focus](Example-Gear#Focus), [Ritual](Ritual), and advanced crafting.
+
+##### Basic Resonant
+Otherwise unusable pieces of damaged artifacts, fetishes, and other superstitious items.
+
+> Shards of broken mirror, shavings of an ancient device
+
+##### Artisan Resonant
+Gemstones and fancy powders.
+
+> Ruby, Sapphire, Diamond shards / stones
+
+
+##### Exotic Resonant
+Intentionally imbued powders, items warped from long exposure to enchanted places.
+
+> Crystallized magic, liquified elements
+
+---
+
+#### Liquid
+Spirits, elixirs, strange fluids, anything that can be used to dissolve material, treat it or as lubrication. Useful in crafting [Comestibles](Comestibles).
```

Comment:
Readability suggestion: "Spirits, elixirs, strange fluids, or anything that can be used to dissolve, treat, or lubricate materials."

Reply:
```suggestion
Spirits, elixirs, strange fluids, or anything that can be used to dissolve, treat, or lubricate materials. Useful in crafting [Comestibles](Comestibles).
```